### ISSUE TITLE
[BUG]  Collections not picked up from DAVx5 4.4

### ISSUE THREAD

**Issue by achrodo** [TechbeeAT/jtxBoard#1457]
**Description**

Collections cannot be added to jtxB from the DAVx5 app. It is possible that something changed in DAVx5 v4.4, as I had been successful adding collections to jtxB previously.

**To Reproduce**

1. Go to hamburger menu and select "Collections"
2. Click on the kebab menu in the upper right and select "Add remote collection (DAVx5)"

**Expected behavior**

It is expected that the user is able to select a collection in DAVx5, and for the jtxBoard app to recognize and display this collection.

**Actual Behavior**

When "Add remote collection (DAVx5)" is selected, the DAVx5 app opens with all its typical functionality. However, there is no clear way to actually *select* an existing collection for jtxBoard to recognize. The user is stuck in DAVx5, and no collection is ever provided back to jtxBoard.

**Device and version**
 
 - Device: Pixel 6a
 - Android version: Android 14 (GrapheneOS stable)
 - jtx Board version: 2.07.08.ose
 - DAVx5 version: 4.4-ose
 - Downloaded from: f-Droid

**Comment by patrickunterwegs** [comment]
Hi @achrodo , 
I'm not sure if I fully understand. 
Basically the action, that DAVx5 gets opened when you click on "Add remote collection (DAVx5)" has not changed since it was created, so it always opened DAVx5 where you could choose which collections you would like to synchronize. 

Please go through this article and check if maybe one of the points fixes the problem for you: 
https://jtx.techbee.at/troubles-with-the-synchronization-see-what-could-go-wrong

**Comment by achrodo** [comment]
Hello @patrickunterwegs ,

I think that what may have changed is something with DAVx5. Indeed, jtxB detects DAVx5 as being installed and opens it. The problem is that there's no way to select/choose a collection. The functionality of the screen in DAVx5 that we are directed to is only particular to the DAVx5 app. In other words, we are dumped into DAVx5's list of collections, but with options only native to DAVx5, and what we would see had we opened DAVx5 on its own and manually navigated to the list of CalDAV collections. There is nothing to 'select' and no way back to jtxB. No additional collections are added to jtxB's "Collections" screen.

I hope that better describes the problem. I'm confident that I'm doing everything else right according to the link you provided. If I have a chance, I may roll back the version of DAVx5 to the previous version and try this again.
I have valid collections for notes/journal and tasks, as I've got Evolution picking them up on the desktop.

I checked, and jtxB is included in the app permissions for DAVx5.

I should note that in the DAVx5 settings, I only see another app, Tasks.org, but I don't see jtxB, and am assuming that I should see that.
